test/lender/Lender.sol

```diff
@@ -225,17 +226,17 @@ contract Lender {
   /// @param d deadline ?
   function lend(uint8 p, address u, uint256 m, uint256 a, uint256 r, address x, address t, uint256 d) public returns (uint256) {
       // Instantiate market and tokens
-      address market = IIlluminate(illuminate).markets(u, m)[p];
-      require(ITempus(market).yieldBearingToken() == IErc20Metadata(u), 'tempus underlying != underlying');
-      require(ITempus(market).maturityTime() == m, 'tempus maturity != maturity');
+      address tempusToken = IIlluminate(illuminate).markets(u, m)[p];
```

Comment:
the whole `address principal` or `address tToken` thing again...

`tToken` follows the pattern that `Element...` set. again, i think we change them all to `principal` at some point...

Reply:
Agree, `principal` is a much better word for this.